Jenkinsfile

```diff
@@ -128,4 +128,13 @@ pipeline {
             }
         }
     }
+    post {
+        always {
+            node('master') {
```

Comment:
any reason why this is not in the PR pipelines?

Reply:
pipeline has error to check out because of lock file before it manages to generate the PR pipeline. PR pipeline is on the build node and we can wipe workspace. The lock issue is here on the master node, it fails to checkout from here

Reply:
has it been tested?

Reply:
still testing out now, I can manually put a lock file there during a build and make sure next build is ok

Reply:
Created lock file manually and verified lock file was deleted after the build finished
```
jenkins@ip-172-31-39-210:~/jobs/brave-core-build-pr-macos/branches/PR-7847/workspace$ touch .git/index.lock
jenkins@ip-172-31-39-210:~/jobs/brave-core-build-pr-macos/branches/PR-7847/workspace$ 


jenkins@ip-172-31-39-210:~/jobs/brave-core-build-pr-macos/branches/PR-7847/workspace$ ls .git
branches  config  description  FETCH_HEAD  HEAD  hooks	index  info  logs  objects  refs  shallow
```
Ran next build https://ci.brave.com/job/brave-core-build-pr-macos/job/PR-7847/5/console without any issue